ibc-primitives/src/traits/proto.rs

```diff
@@ -31,3 +31,14 @@ where
     <Self as TryFrom<P>>::Error: Display,
 {
 }
+
+/// Convenient trait for converting types to a raw Protobuf `Vec<u8>`.
+pub trait ToVec {
```

Comment:
This seems like such a fundamental trait that I wonder if it should be exported as part of the `prelude`?